workbench/public/components/QueryResults/QueryResultsBody.tsx

```diff
@@ -127,15 +130,16 @@ class QueryResultsBody extends React.Component<QueryResultsBodyProps, QueryResul
       selectedItemData: {},
       navView: false,
       isPopoverOpen: false,
-      isDownloadPopoverOpen: false
+      isDownloadPopoverOpen: false,
+      isModalVisible: false,
+      downloadErrorModal: {}
     };
 
     this.expandedRowColSpan = 0;
     this.items = [];
     this.columns = [];
     this.panels = [];
```

Comment:
This line seems unnecessary

Reply:
> Why is it necessary to have separate SQLPage and PPLPage components? Right now they seem to be the same component with different prop/state variable names

It's because we will have different components in SQL and PPL pages in our latest ux design, but not refracted in this PR yet. We will elaborate the details later.

Reply:
> This line seems unnecessary

Removed this line, thanks!